BedrockServer.cpp

```diff
@@ -1989,6 +2000,7 @@ bool BedrockServer::_upgradeDB(SQLite& db) {
     if (db.getUncommittedQuery().empty()) {
         db.rollback();
     }
+    SDEBUG("finished running _upgradeDB()");
```

Comment:
function name in this log line is not needed, it's already logged in every log line.